crucible/from-file-json-manipulation-api/README.md

```diff
@@ -0,0 +1,24 @@
+This example has an `oslat.json` template that the `run.sh` script
```

Comment:
I'd rename the dir to run-file-api to 1) keep it short 2) use 'run-file' name everywhere